frontend/app/templates/components/encryptable/row.hbs

```diff
@@ -40,58 +40,75 @@
       <LinkTo id="encryptable-row-title" class="d-flex align-items-center text-left col-12 col-md-3 col-lg-3
       col-xl-4 underline-hover break-words account-link" @route="encryptables.show" @model={{@encryptable.id}}>
         {{@encryptable.name}}
       </LinkTo>
-      {{#if (and (lte getAttributesAmount 2) (gt getAttributesAmount 0))}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "username")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="username" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "password")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block mr-0 ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="password" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "email")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block mr-0 ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="email" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "token")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block mr-0 ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="token" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "pin")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block mr-0 ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="pin" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
-        {{#if (is-included @encryptable.usedEncryptedDataAttrs "custom_attr")}}
-          <div class="encryptable-container d-none d-md-block d-lg-block d-xl-block mr-0 ml-0 mr-md-2 ml-md-5 mr-lg-2 ml-lg-5 mr-xl-2 ml-xl-2">
-            <Encryptable::AttributeField @attribute="customAttr" @row={{true}} @encryptable={{@encryptable}}
-                                         @visibleByDefault={{false}}/>
-          </div>
-        {{/if}}
+      {{#if this.isTransferredCredentials }}
```

Comment:
```suggestion
      {{#if this.wasSent }}
```

```suggestion
      {{#if this.hasSender }}
```

Reply:
does it make sense to show this information outside inbox folder?

so if someone moves the received encryptable to another folder ... maybe we can just hide it?

Reply:
In my view, the renaming is not needed, because the variable name says, what value is saved in it?!